packages/framework-core/src/booster-token-verifier.ts

```diff
@@ -54,12 +55,20 @@ class TokenVerifierClient {
         if (err) {
           return reject(err)
         }
-        return resolve(this.tokenToUserEnvelope(decoded))
+        const jwtToken = decoded as any
+        if (this.tokenVerifierConfig?.extraValidation) {
+          try {
+            this.tokenVerifierConfig?.extraValidation(jwtToken?.header, jwtToken?.payload)
```

Comment:
@javiertoledo Regarding your questions:

* Is there anything in the token apart from the header and the payload? As far as I know, the information is divided into three sections header, payload, and signature. I think we could include them.
* Would it make sense to inject the raw token too? Umm, maybe if the user wants to use the raw token for an external issuer validation or whatever reason it could make sense.

Reply:
Fixed in d43ea9cb9848e447a39abaa483e34388613d0524